bugsnag-plugin-android-ndk/detekt-baseline.xml

```diff
@@ -3,6 +3,7 @@
   <ManuallySuppressedIssues/>
   <CurrentIssues>
     <ID>ComplexMethod:NativeBridge.kt$NativeBridge$override fun onStateChange(event: StateEvent)</ID>
+    <ID>LongMethod:EventMigrationV4Tests.kt$EventMigrationV4Tests$@Test fun testMigrateEventToLatest()</ID>
```

Comment:
This seemed like a sensible long method, since it verifies several things and pulling the objects out of the test scope didn't make sense. Though there will be another "issue" for every test in this format. 🤷